spec/support/inputs_outputs_pairs.rb

```diff
@@ -11,90 +11,90 @@ module JsonApi::Parameters::Testing
       ],
       'single root, single relationship, multiple related resources' => [
         {
-         :data => {
-           :type => "contract",
-           :attributes => {
-             :total_amount=>"40.0",
-             :start_date=>"2018-10-18"
-           },
-           :relationships=>{
-             :products=>{
-               :data=>[
-                 {
-                   :id=>"4", :type=>"product"
-                 }, {
-                   :id=>"5", :type=>"product"
-                 }, {
-                   :id=>"6", :type=>"product"
-                 }
-               ]
-             },
-             :customer=>{
-               :data=>{
-                 :id=>"1", :type=>"customer"
-               }
-             }
-           }
-         },
-         :included=>[
-           {
-             :id=>"4",
-             :type=>"product",
-             :attributes=>{
-               :category=>"first_category",
-               :amount=>"15.0",
-               :name=>"First product",
-               :note=>""
-             }
-           },
-           {
-             :id=>"5",
-             :type=>"product",
-             :attributes=>{
-               :category=>"first_category",
-               :amount=>"10.0",
-               :name=>"Second Product",
-             }
-           },
-           {
-             :id=>"6",
-             :type=>"product",
-             :attributes=>{
-               :category=>"second_category",
-               :amount=>"15.0",
-               :name=>"Third Product",
-             }
-           }
-         ]
-       },
-       {
-         contract: {
-           start_date: '2018-10-18',
-           total_amount: '40.0',
-           customer_id: '1',
-           products_attributes: [
-             {
-               id: '4',
-               :category=>"first_category",
-               :amount=>"15.0",
-               :name=>"First product",
-               note: ''
-             },
-             {
-               id: '5',
-               :category=>"first_category",
-               :amount=>"10.0",
-               :name=>"Second Product"
-             },
-             {
-               id: '6',
-               :category=>"second_category",
-               :amount=>"15.0",
-               :name=>"Third Product"
-             }
-           ]
-         }
-       }
+          :data => {
```

Comment:
Could you convert all hashes to a new format (`data: {}` instead of `:data => {}`) to have the code more consistent?

Reply:
Yeah I've been trying to, intellij idea however has some trouble doing it automatically, probably because its a mixed key values hash.

Reply:
I'll do it manually.